server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -0,0 +1,255 @@
+import path = require('path')
+import { Workspace } from '@aws/language-server-runtimes/out/features'
+import * as fs from 'fs'
+import * as os from 'os'
```

Comment:
We have a `fs` wrapper in the `workspace` feature that will allow you to do this in a cross platform way:

https://github.com/aws/language-server-runtimes/blob/main/src/features/workspace.ts#L21

Reply:
Same goes for `path`

Reply:
Do you know whats the best way to use `path` which is cross platform ?